src/components/POLine/POLineInvoices/POLineInvoices.js

```diff
@@ -0,0 +1,85 @@
+import React from 'react';
+import { FormattedMessage } from 'react-intl';
+import PropTypes from 'prop-types';
+import { Link } from 'react-router-dom';
+
+import {
+  find,
+  get,
+} from 'lodash';
+
+import { MultiColumnList } from '@folio/stripes/components';
+import { AmountWithCurrencyField } from '@folio/stripes-acq-components';
+
+import FolioFormattedDate from '../../FolioFormattedDate';
+
+const POLineInvoices = ({ lineInvoices, vendors, quantityPhysical, quantityElectronic, pieces }) => {
+  const captionsList = pieces.map(item => item.caption).join(', ');
+  const resultFormatter = {
+    'invoice': invoice => (
+      <Link
+        data-test-link-to-invoice
+        to={`/invoices/view/${invoice.id}`}
+      >
+        {get(invoice, 'folioInvoiceNo', '')}
+      </Link>
+    ),
+    'invoiceDate': invoice => <FolioFormattedDate value={get(invoice, 'invoiceDate')} />,
+    'vendorName': invoice => get(find(vendors, ['id', get(invoice, 'vendorId', '')]), 'name', ''),
+    'status': invoice => get(invoice, 'status', ''),
+    'quantity': () => quantityPhysical + quantityElectronic,
+    'expendedAmount': invoice => (
+      <AmountWithCurrencyField
+        currency={get(invoice, 'currency')}
+        amount={get(invoice, 'total', 0)}
+      />
+    ),
+    'pieces': () => captionsList,
```

Comment:
this looks a bit weird

Reply:
In MCL we have 'Piece(s)'column, but actually, it's a list of pieces captions

Reply:
shouldn't it be `pieces: captionsList,` ?

Reply:
the same list for each line?

Reply:
right, I was thinking of `captionsList` as a function. However, it makes not much sense to display the same value for all rows. maybe you need to clarify requirements rather than implement it first.

Reply:
Dennis [comment](https://issues.folio.org/browse/UIOR-123?focusedCommentId=55470&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-55470) about qty and pieces. 
`pieces: captionsList` wouldn't work